### PR TITLE
has a ding now

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: lib/console.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ module.exports = function(options, tags, message) {
   if (typeof message === 'object') {
     message = stringify(message, null, (options.pretty) ? '  ' : '').replace(/\\n/g, '\n');
   }
+  if (options.consoleBell) {
```
Comment (jgallen23) [inline lib/console.js]: this should only happen if certain tags are passed

**Comment by jgallen23** [comment]
the ding should only happen if certain tags happen.  this would be very annoying otherwise

**Comment by orthagonal** [comment]
now supports the 'ding' tag

**Comment by orthagonal** [comment]
@jgallen23:
-dings on 'error' by default
-lets you optionally pass in a list of tags that will trigger a ding 
-updated tests that include the 'error' tag to be correct

File: test/logr.test.js [hunk 2]
```diff
@@ -37,6 +37,18 @@ describe('logr', () => {
       log(['tag1', 'tag2'], 'message');
       expect(lastMessage).to.contain('[tag1,tag2] message');
     });
+    it('allows you to specify a tag to trigger a ding', () => {
```
Comment (jgallen23) [inline test/logr.test.js]: need to add a test case for `consoleBell: false`

File: lib/console.js [hunk 3]
```diff
@@ -32,6 +32,14 @@ module.exports = function(options, tags, message) {
...
+    for (let i = 0; i < tags.length; i++) {
```
Comment (jgallen23) [inline lib/console.js]: is there a lodash method that would clean this up?

File: test/logr.test.js (hunk 2 above)
Comment (jgallen23) [inline test/logr.test.js]: also a test case to make sure it isn't added if it doesn't match.  there is already an indirect test, but would be nice to have an explicit one

**Comment by orthagonal** [comment]
@jgallen23 : finished the above-mentioned tasks

**Comment by orthagonal** [comment]
@jgallen23 
-  tags are now optional, log("test") and log({test: 123}) work as expected

**Pull request merged**